bot/service/governance_service.py

```diff
@@ -5,60 +5,32 @@
 import requests
 from telegram.utils.helpers import escape_markdown
 
-from constants.constants import LCD_URL
-from constants.messages import NETWORK_ERROR_MSG
+from constants.constants import LCD_ENDPOINT, TERRA_STATION_URL
+from constants.env_variables import NETWORK
 
 
 def get_governance_proposals(params=None) -> List:
-    """
-    Return all governance proposals
-    """
-
-    response = requests.get(f'{LCD_URL}gov/proposals', params=params)
+    response = requests.get(f'{LCD_ENDPOINT}gov/proposals', params=params)
```

Comment:
I specified an IP as LCD Endpoint and get this error (because the IP doesn't have an ending slash) 
```
requests.exceptions.MissingSchema: Invalid URL '153.135.223.241gov/proposals': No schema supplied. Perhaps you meant http://153.135.223.241gov/proposals?
```

Should be either specified very clearly in the README or add the slash here

Reply:
and http:// is also missing